.travis.yml

```diff
@@ -27,10 +27,15 @@ install: mvn install -q -DskipTests=true
 
 script: mvn test -q
 
+# Bucket-type to activate before running Riak KV tests.
```

Comment:
Does a new bucket type need to be created? Is there a default bucket type that can be used for tests?